src/components/pages/Offer/OfferDetails/__specs__/OfferEdition.spec.jsx

```diff
@@ -202,5 +204,250 @@ describe('offerDetails - Edition', () => {
       const newTitleValue = await screen.findByDisplayValue('Mon nouveau titre')
       expect(newTitleValue).toBeInTheDocument()
     })
+
+    it('non-sync offers should have some read-only fields', async () => {
```

Comment:
```suggestion
    it('should allow edition of editable fields only', async () => {
```
Pour coller avec le `describe` qui est avant